src/main/java/com/google/sps/data/Email.java

```diff
@@ -74,6 +77,34 @@ public static String addNewPost(Post post) throws IOException {
     return emailContent;
   }
 
+  /**
+    * Add all ranked posts into daily digest email.
+    *
+    * @param posts ranked from a college
+    * @return string of HTML file with 3 ranked posts
+    * @throws IOException
+    */
+  public static String addRankedPosts(ArrayList<Post> rankedPosts) throws IOException {
+
+    String emailContent = getStringFromHTML(dailyDigestPath);
+
+    // Split up email content to inject multiple posts in between.
+    int split = emailContent.indexOf("</p>") + 4;
+    String firstPartOfEmail = emailContent.substring(0, split);
+    String lastPartOfEmail = emailContent.substring(split);
+    
+    // Add all ranked posts information to email content.
```

Comment:
You can give an example of what this looks like e.g. `Santa Clara University @ XXX | 10:00 - 12:12` so readers do not have to  spend time running through this to know what it does.

Reply:
Will add!